easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-0.5.1b-intel-2018a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'RAxML-NG'
+version = '0.5.1b'
+
+homepage = 'https://github.com/amkozlov/raxml-ng'
+description = """RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality
+    criterion. Its search heuristic is based on iteratively performing a series of Subtree Pruning and
+    Regrafting (SPR) moves, which allows to quickly navigate to the best-known ML tree."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/v%(version)s']
```

Comment:
@ajaust Download URL isn't correct now, the trailing `b` in the version is not included in the download URL...

Proposal:
```python
source_urls = ['https://github.com/amkozlov/raxml-ng/releases/download/v%s' % version[:-1]]